src/sentry/api/endpoints/project_transaction_threshold_override.py

```diff
@@ -0,0 +1,161 @@
+        if len(projects) != 1:
+            raise ParseError("Only 1 project per transaction threshold")
+
+        return projects[0]
+
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return self.respond(status=status.HTTP_404_NOT_FOUND)
+
+        project = self.get_project(request, organization)
+
+        try:
+            project_threshold = ProjectTransactionThresholdOverride.objects.get(
+                transaction=request.GET.get("transaction"),
+                project_id=project.id,
+                organization_id=organization.id,
+            )
+        except ProjectTransactionThresholdOverride.DoesNotExist:
+            return Response(
+                data={
+                    "transaction": request.GET.get("transaction"),
+                    "projectId": str(project.id),
+                    **DEFAULT_THRESHOLD,
+                },
+                status=status.HTTP_200_OK,
+            )
+
+        return Response(
+            serialize(
+                project_threshold,
+                request.user,
+            ),
+            status.HTTP_200_OK,
+        )
+
+    def post(self, request, organization):
+        if not self.has_feature(organization, request):
+            return self.respond(status=status.HTTP_404_NOT_FOUND)
+
+        project = self.get_project(request, organization)
+        serializer = ProjectTransactionThresholdOverrideSerializer(
+            data=request.data,
+            context={
+                "organization": organization,
+                "project": project,
+            },
+        )
+
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        data = serializer.validated_data
+
+        with transaction.atomic():
+            (
+                transaction_threshold,
+                created,
+            ) = ProjectTransactionThresholdOverride.objects.update_or_create(
+                transaction=data["transaction"],
+                project_id=project.id,
+                organization_id=organization.id,
+                defaults={
+                    "threshold": data["threshold"],
+                    "metric": data["metric"],
+                    "edited_by": request.user,
+                },
+            )
+
+        return Response(
+            serialize(
+                transaction_threshold,
+                request.user,
+            ),
+            status=status.HTTP_201_CREATED if created else status.HTTP_200_OK,
+        )
+
+    def delete(self, request, organization):
+        if not self.has_feature(organization, request):
+            return self.respond(status=status.HTTP_404_NOT_FOUND)
+
+        project = self.get_project(request, organization)
+        data = request.data
+        try:
+            transaction_threshold = ProjectTransactionThresholdOverride.objects.get(
+                transaction=data["transaction"],
```

Comment:
Does this need to validate `"transaction"` is a valid key to avoid a `KeyError`?